### DISCUSSION TITLE
graphql/apollo-angular/docs/get-started

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/apollo-angular#1859] [Docs Discussions]
# graphql/apollo-angular/docs/get-started

Set up Apollo in your Angular app

https://the-guild.dev/graphql/apollo-angular/docs/get-started

**Comment by Aaditya-Rane** [comment]
Page starting link "Apollo Angular" is redirecting to "https://www.nasa.gov/mission_pages/apollo/missions/index.html"

**Comment by JanRou** [comment]
Thank you for a good get started page for Angular Apollo GraphQL client! The *ng add apollo-angular* works very good for a newcomer. It generates code for the latest Angular 17. The code on this page how ever seems out of date. That's include the Stackblitz sample. App.module.ts doesn't exist anymore and is replaced by  a *graphqlProvider*. It looks like a breaking change.
Where can I find a migration guide or like?
